cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -625,8 +624,15 @@ protected void startProgram(ProgramId program, int expectedStatusCode)
                                 program.getVersion(),
                                 program.getType().getCategoryName(),
                                 program.getProgram());
-    HttpResponse response = doPost(getVersionedAPIPath(path, program.getNamespace()),
-                                   GSON.toJson(ImmutableMap.<String, String>of()));
+    startProgram(path, program.getNamespace(), ImmutableMap.<String, String>of(), expectedStatusCode);
+  }
+
+  /**
+   * Tries to start the given program with the given runtime arguments and expect the call completed with the status.
+   */
+  protected void startProgram(String path, String namespaceId, Map<String, String> args, int expectedStatusCode)
```

Comment:
It seems in this file, the style which is used breaks lines at exceptions: https://github.com/caskdata/cdap/blob/80a3162de4b6d7871fd25ed3a98c102c03aac5ca/cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/AppFabricTestBase.java#L621

Should I follow the existing style in the file or should I just follow your guidelines?